src/installation.md

```diff
@@ -51,4 +51,6 @@ sudo reboot
 
 ## Step 8: Launched Sugar
 
-Once the Raspberry Pi reboots, Sugar should start automatically. If not you may have to follow up with troubleshoots.
\ No newline at end of file
+Once the Raspberry Pi reboots, Sugar should start automatically. 
+Dont Forget to check if the Sugar Desktop Environment is installed , and turning off auto logging in RPi-config settings. 
```

Comment:
Don't forget to check if the Sugar Desktop Environment is installed. And don't forget to turn off auto-logging in the RPi-config settings.

Reply:
Yes there is a typing error it should say what you have written.